src/trackingModules/modeling/ModelingTracking.js

```diff
@@ -0,0 +1,68 @@
+export default class ModelingTracking {
+  constructor(eventBus, bpmnJSTracking, elementRegistry, selection) {
+    this._eventBus = eventBus;
+    this._bpmnJSTracking = bpmnJSTracking;
+    this._elementRegistry = elementRegistry;
+    this._selection = selection;
+
+    this._eventBus.on(
+      'commandStack.elements.create.postExecuted',
+      this.trackElementsCreate.bind(this)
+    );
+
+    this._eventBus.on(
+      'commandStack.shape.append.postExecuted',
+      this.trackElementAppend.bind(this)
+    );
+
+    this._eventBus.on(
+      'commandStack.shape.replace.postExecuted',
+      this.trackElementReplace.bind(this)
+    );
+  }
+
+  trackElementsCreate(e) {
+    const { context } = e;
+
+    this._bpmnJSTracking.track({
+      name: 'modeling.createElement',
+      data: {
+        elements: context.elements
+      }
```

Comment:
As we track `createElements` (plural), and the data is an array, should the event name reflect that?

Reply:
It definitely should!